.github/workflows/opcua-plc.yml

```
@@ -45,5 +45,76 @@ jobs:
           go_version: ${{ env.GO_VERSION }}
       - name: Install Ginkgo
         run: go install github.com/onsi/ginkgo/v2/ginkgo@v2.19.0
+      - name: Install Tcping
+        run: go install github.com/cloverstd/tcping@latest
+      - name: Check S7 port availability
+        id: check_s7
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:28: Double quote to prevent globbing and word splitting

(shellcheck)